src/dal/lucky.ts

```diff
@@ -0,0 +1,29 @@
+import { database } from "../libs/database";
+import { getRandomNumberBetweenTwoNumbers } from "../randomizer";
+import { numbersSignsBG_zodiacDirBg } from "../models/signs";
+
+export async function getLuckyTextByRandomNumber(): Promise<string> {
+  const luckyNumber = getRandomNumberBetweenTwoNumbers(1, 12);
+  // Note! Sign names up to https://zodiac.dir.bg
+  const signObj = numbersSignsBG_zodiacDirBg.find(element => element.id === luckyNumber);
```

Comment:
I see business logic here and I don't want to. This is DAL - Data Access Layer, which means that I want to see only a call to the database and return of result here. All of the business logic from here should be extracted into a separate function in services folder.